eoscostarica/eosio-dashboard
# Home composition

### Refactor UI

### What does this PR do?

- Resolve #405 #224 

### Steps to test

1. run `make run lacchain`
2. go to localhost:3000
3. see home page

#### CheckList
